hsctl

```diff
@@ -243,18 +244,23 @@ loaddb_hs() {
 
 migrate_hs() {
     echo "MIGRATE:"
-    echo "  - docker exec -u hydro-service hydroshare python manage.py migrate sites --noinput"
-    docker exec -u hydro-service hydroshare python manage.py migrate sites --noinput
-    echo "  - docker exec -u hydro-service hydroshare python manage.py migrate --fake-initial --noinput"
-    docker exec -u hydro-service hydroshare python manage.py migrate --fake-initial --noinput
-    echo "  - docker exec -u hydro-service hydroshare python manage.py fix_permissions"
-    docker exec -u hydro-service hydroshare python manage.py fix_permissions
+    echo "  - docker exec -u hydro-service hydroshare python -W ignore::DeprecationWarning:: manage.py migrate sites --noinput"
+    docker exec -u hydro-service hydroshare python -W ignore::DeprecationWarning:: manage.py migrate sites --noinput
+    echo "  - docker exec -u hydro-service hydroshare python -W ignore::DeprecationWarning:: manage.py migrate --fake-initial --noinput"
+    docker exec -u hydro-service hydroshare python -W ignore::DeprecationWarning:: manage.py migrate --fake-initial --noinput
+    echo "  - docker exec -u hydro-service hydroshare python -W ignore::DeprecationWarning:: manage.py fix_permissions"
+    docker exec -u hydro-service hydroshare python -W ignore::DeprecationWarning:: manage.py fix_permissions
 }
 
 manage_py_hs() {
     echo "MANAGEPY:"
-    echo "  - docker exec -u hydro-service -ti hydroshare python manage.py ${@:2}"
-    docker exec -u hydro-service -ti hydroshare python manage.py ${@:2}
+    if [ "$2" == "--verbose" ]; then
```

Comment:
This expects --verbose to always be the second argument, and just ignores any other argument input.  We should probably validate arguments passed in and display help if it's bad.  I'm in favor of using switch/case to setup a nice pattern in case we add more arguments.

Reply:
Yeah, I forgot to update that to `${@:3}`

However the 2nd argument only thing is something I just coped from below in the same file for --db. I would love for hsctl to be great when it comes to arg parsing but the --verbose flag is really just something probably *I* am gonna be running as I upgrade to 1.11... so I don't know if it's dimiishing returns doing the extra work to make --verbose work as any argument

Reply:
Oh wait, no... I did update it to ${@:3}

Reply:
ok, we just need to be careful if we eventually do add more arguments to follow a maintainable pattern.

Reply:
I like your switch / case idea in the other comment. That looks like a nice pattern that we can move to.